qiskit_experiments/visualization/plotters/base_plotter.py

```diff
@@ -66,47 +67,51 @@ class BasePlotter(ABC):
     plotter.
 
     .. note::
-        If a figure-option called "foo" is not set in the drawers figure-options (:attr:`~BaseDrawer.
-        figure_options`), but is set in the plotters figure-options (:attr:`figure_options`), it will
-        not be copied over to the drawer when the :meth:`figure` method is called. This means that some
-        figure-options from the plotter may be unused by the drawer. :class:`BasePlotter` and its
-        subclasses filter these options before setting them in the drawer as subclasses of
-        :class:`BaseDrawer` may add additional figure-options. To make validation easier and the code
-        cleaner, the :meth:`figure` method conducts this check before setting figure-options in the
-        drawer.
-
-    Example:
-        .. code-block:: python
-            plotter = MyPlotter(MyDrawer())
-
-            # MyDrawer contains the following figure_options with default values.
-            plotter.drawer.figure_options.xlabel
-            plotter.drawer.figure_options.ylabel
-
-            # MyDrawer does NOT contain the following figure-option
-            # plotter.drawer.figure_options.unknown_variable    # Raises an error as it does not exist in
-                                                                # `plotter.drawer`.
-
-            # If we set the following figure-options, they will be set in the drawer.
-            plotter.set_figure_options(xlabel="Frequency", ylabel="Fidelity")
-
-            # During a call to `plotter.figure()`, the drawer figure-options are updated.
-            # The following values would be returned from the drawer.
-            plotter.drawer.figure_options.xlabel                # returns "Frequency"
-            plotter.drawer.figure_options.ylabel                # returns "Fidelity"
-
-            # If we set the following option and figure-option will NOT be set in the drawer.
-            plotter.set_options(plot_fit=False)                 # Example plotter option
-            plotter.set_figure_options(unknown_variable=5e9)    # Example figure-option
-
-            # As `plot_fit` is not a figure-option, it is not set in the drawer.
-            plotter.drawer.options.plot_fit     # Would raise an error if no default exists, or return a
-                                                # different value to `plotter.options.plot_fit`.
-
-            # As `unknown_variable` is not set in the drawers figure-options, it is not set during a call
-            # to the `figure()` method.
-            # plotter.drawer.figure_options.unknown_variable    # Raises an error as it does not exist
-                                                                # in `plotter.drawer.figure_options`.
+        If a figure-option called "foo" is not set in the drawers figure-options
+        (:attr:`BaseDrawer.figure_options`), but is set in the plotters figure-options
```

Comment:
```suggestion
        (:attr:`BaseDrawer.figure_options`) but is set in the plotter's figure options
```

Reply:
Addressed in afb8a57, e79bdc1, and a4685ca.